core/validation/initial_validation/schemas.py

```diff
@@ -105,3 +105,31 @@
         '"Future_High_Street_Fund".',
     ),
 ]
+# TODO replace the placeholder error messages with those from design
+PF_ROUND_1_INIT_VAL_SCHEMA = [
+    BasicCheck(
+        sheet="Metadata",
+        column=0,
+        row=0,
+        expected_values=("1",),
```

Comment:
Do we know what the actual reporting period will be?

Reply:
There is a row in the Start tab which has a select dropdown with all the reporting periods - for example Q3 Oct - Dec 23/24. We can use this to validate against instead of the reporting round?

Reply:
My bad, mixed up reporting period and round - this is fine for now